README.md

```diff
@@ -320,13 +320,21 @@ setState('iot.0.app.priority', 'normal'); // optional. Priority: 'high' or 'norm
 setState('iot.0.app.title', 'ioBroker'); // optional. Default "ioBroker"
 setState('iot.0.app.message', 'Message text'); // important, that ack=false (default)
 
-// or just one state
+// or just one state (this also allows to use payload -> `actions` and `link` property)
 // only is message is mandatory. All other are optional
 setState('iot.0.app.message', JSON.stringify({
   message: 'Message text',
   title: 'ioBroker',
   expire: 60,
-  priority: 'normal'
+  priority: 'normal', 
+  payload: {
```

Comment:
I would add an "optional" mark in the documentation for every attribute that is not required

Reply:
I thought about it but there is a comment above all from you which is sufficient or?

Line 324
> // only is message is mandatory. All other are optional